Lib/test/test_bz2.py

```diff
@@ -76,11 +77,14 @@ class BaseTest(unittest.TestCase):
     BIG_DATA = bz2.compress(BIG_TEXT, compresslevel=1)
 
     def setUp(self):
-        self.filename = support.TESTFN
+        fd, self.filename = tempfile.mkstemp()
+        os.close(fd)
 
     def tearDown(self):
-        if os.path.isfile(self.filename):
+        try:
             os.unlink(self.filename)
+        except FileNotFoundError:
```

Comment:
You can use `unlink` from `test.support`. It already catches `FileNotFoundError` and works fine on Windows. (This comment also applies to https://github.com/python/cpython/commit/6a62e1d365934de82ff7c634981b3fbf218b4d5f.)

https://github.com/python/cpython/blob/master/Lib/test/support/__init__.py#L410-L414